2023-08-15T00:21:32.1547053Z ##[group]Run actions/setup-python@v4
2023-08-15T00:21:32.1547408Z with:
2023-08-15T00:21:32.1547701Z   python-version: 3.8
2023-08-15T00:21:32.1548021Z   architecture: x64
2023-08-15T00:21:32.1548313Z   cache: pip
2023-08-15T00:21:32.1548557Z   check-latest: false
2023-08-15T00:21:32.1549041Z   token: ***
2023-08-15T00:21:32.1549360Z   update-environment: true
2023-08-15T00:21:32.1549694Z   allow-prereleases: false
2023-08-15T00:21:32.1549956Z ##[endgroup]
2023-08-15T00:21:32.3300440Z ##[group]Installed versions
2023-08-15T00:21:32.3389786Z Successfully set up CPython (3.8.17)
2023-08-15T00:21:32.3390888Z ##[endgroup]
2023-08-15T00:21:33.1759425Z [command]/opt/hostedtoolcache/Python/3.8.17/x64/bin/pip cache dir
2023-08-15T00:21:33.6744697Z /home/runner/.cache/pip
2023-08-15T00:21:34.0553320Z Received 6339595 of 6339595 (100.0%), 36.2 MBs/sec
2023-08-15T00:21:34.0554411Z Cache Size: ~6 MB (6339595 B)
2023-08-15T00:21:34.0584276Z [command]/usr/bin/tar -xf /home/runner/work/_temp/45f465e2-69a1-4883-96d2-a5d1a9885bcc/cache.tzst -P -C /home/runner/work/GitHub_Stats_A/GitHub_Stats_A --use-compress-program unzstd
2023-08-15T00:21:34.0804269Z Cache restored successfully
2023-08-15T00:21:34.0823357Z Cache restored from key: setup-python-Linux-22.04-Ubuntu-python-3.8.17-pip-60f2a6c0212a9b74a155cd81b58dd07fd8b180833132566260d95f4de42fe218
